tests/integration_tests/scheduler/test_openpbs_driver.py

```diff
@@ -107,3 +108,39 @@ def test_openpbs_driver_with_poly_example():
         "--enable-scheduler",
         "poly.ert",
     )
+
+
+@pytest.mark.timeout(180)
+@pytest.mark.integration_test
+@pytest.mark.usefixtures("copy_poly_case")
+def test_openpbs_driver_with_poly_example_failing_submit_fails_ert_and_propagates_exception_to_user(
+    monkeypatch, caplog
+):
+    monkeypatch.setattr(OpenPBSDriver, "submit", lambda *args, **kwargs: 1 / 0)
+    with open("poly.ert", mode="a+", encoding="utf-8") as f:
+        f.write("QUEUE_SYSTEM TORQUE\nNUM_REALIZATIONS 2")
+    with pytest.raises(ErtCliError):
+        run_cli(
+            ENSEMBLE_EXPERIMENT_MODE,
+            "--enable-scheduler",
+            "poly.ert",
+        )
+    assert "ZeroDivisionError" in caplog.text
+
+
+@pytest.mark.timeout(60)
+@pytest.mark.integration_test
+@pytest.mark.usefixtures("copy_poly_case")
+def test_openpbs_driver_with_poly_example_failing_poll_fails_ert_and_propagates_exception_to_user(
+    monkeypatch, caplog
+):
+    monkeypatch.setattr(OpenPBSDriver, "poll", lambda *args, **kwargs: 1 / 0)
```

Comment:
Ditto... Don't actually submit jobs to the cluster if all we care about is that `poll` should fail.